mitmproxy/web/static/app.css

```diff
@@ -163,6 +163,12 @@ header .menu {
   max-height: 500px;
   overflow-y: auto;
 }
+.btn-option {
+  margin: 2px 2px 2px 2px;
+}
+.text-padding {
+  padding: 0px 5px 0px 5px;
+}
```

Comment:
We should make sure that we don't have those super-generic CSS classes. I think it's just fine to have this specific, e.g. `.menu .btn`.